en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -71,13 +72,13 @@ The figure below shows how you can access your remote receiver (please find the
 ## Spektrum Satellite Receivers
 
 The V5 nano has a dedicated DSM cable.
-The Spektrum satellite receiver should be connected to the flight control DSM/SBUS/RSSI interface.
+The Spektrum satellite receiver should be connected to the flight control DSM/SBUS/RSSI interface. 
```

Comment:
@cuhome - Do you mean "the Spektrum" receiver (ie a particular one supplied) or "a Spektrum" receiver (i.e. if you are using a spektrum receiver, this is where it would be attached)?